tests/integration/flow.yml

```diff
@@ -0,0 +1,8 @@
+jtype: Flow
+version: '1'
```

Comment:
this integration flow will need to add the `py_modules`

Reply:
updated